junit-5/src/test/java/utils/MarkSessionStatus.java

```diff
@@ -0,0 +1,11 @@
+package utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+
+public class MarkSessionStatus {
+    public void markTestStatus(String status, String reason, WebDriver driver){
+        JavascriptExecutor jse = (JavascriptExecutor) driver;
+        jse.executeScript("browserstack_executor: {\"action\": \"setSessionStatus\", \"arguments\": {\"status\": \""+ status + "\", \"reason\": \"" + reason + "\"}}");
```

Comment:
Possible to use a hashmap instead?

```
{\"action\": \"setSessionStatus\", \"arguments\": {\"status\": \""+ status + "\", \"reason\": \"" + reason + "\"}}"
```

Reply:
Should I replace the JSE inputs with a hashmap? Need more details on the requirement.

Reply:
i meant a json object, providing a direct string is making this difficult to read. we can later convert it from object to string

Reply:
The syntax used is to keep it simple and inline with our [docs](https://www.browserstack.com/docs/automate/selenium/getting-started/java#mark-tests-as-passed-or-failed).

Reply:
Using REST API instead of JSE